packages/node_modules/@webex/internal-plugin-search/test/integration/spec/search.js

```diff
@@ -214,6 +263,24 @@ describe('plugin-search', () => {
         }
       },
       // Single word
+      // Asserts comments
+      // Includes remote clusters
+      {
+        given: {
+          user: 'spock',
+          query: 'hElLo',
+          includeRemoteClusterReferences: true
+        },
+        expected: {
+          path: 'object.displayName',
+          results: [
+            {name: 'Hello, Spock.', count: 3},
+            {name: 'Hello, Doctor.', count: 3}
+          ],
+          resultsCount: 6
```

Comment:
Did you add 2 additional activities? and that's what we are asserting on? It's hard for me to follow the test flow.

Reply:
yeah i added 2 more tests. i will add 1 more to test if we get back no results from remote clusters. 

The test creates 2 convos and 1 eu convo, and for each of these convos there's a "Hello, Spock." and "Hello, Doctor." message so if user searched for "hello", there would be 6 total messages. Since there are 3 convos, there are 3 "Hello, Spock." results and 3 "Hello, Doctor." results (in the case of including remote clusters). If we specified `includeRemoteClusterReferences: false`, there there would only be 4 total results (2 spocks and 2 doctor messages) because it wouldn't return the eu convo messages